oneview_redfish_toolkit/blueprints/ethernet_interface.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from flask import Blueprint
+from flask import g
+from flask_api import status
+from werkzeug.exceptions import abort
+
+from oneview_redfish_toolkit.api.ethernet_interface import EthernetInterface
+from oneview_redfish_toolkit.blueprints.util.response_builder import \
+    ResponseBuilder
+
+ethernet_interface = Blueprint("ethernet_interface", __name__)
+
+
+@ethernet_interface.route(
+    "/redfish/v1/Systems/<server_profile_uuid>/EthernetInterfaces/<eth_id>",
+    methods=["GET"])
+def get_ethernet_interface(server_profile_uuid, eth_id):
```

Comment:
I think no because this method is mapped by a route http, it should not be called directly, but if you think is necessary, I can put comments to the args.